java/demo/src/com/google/phonenumbers/PhoneNumberParserServlet.java

```diff
@@ -214,6 +217,29 @@ private StringBuilder getOutputForSingleNumber(
       output.append("</TABLE>");
       output.append("</DIV>");
 
+      boolean isPossibleShort = shortInfo.isPossibleShortNumber(number);
```

Comment:
Won't it be more confusing if short number info shows for some numbers but not others? Okay, I can wrap this in a test for validity.